### DISCUSSION TITLE
How about using the `image` crate?

### DISCUSSION THREAD

**Discussion by AzimMuradov** [hecrj/iced#950] [Q&A] [answered]
[Image crate](https://github.com/image-rs/image) has fantastic support for a variety of image formats. I think it would be easier in the future to implement image related functionality if iced was using it (maybe since 1.0.0?).

**Comment by AzimMuradov** [comment]
I can also try to implement this and make pr if you think it's a good idea.

**Comment by hecrj** [comment, marked as answer]
It's already being used internally by `iced_wgpu` whenever the `image` feature is enabled:

https://github.com/hecrj/iced/blob/4994d34abab3222f9a8fd7a9a3e63f969ca97ffc/wgpu/Cargo.toml#L50-L54

The main reason `iced_native` does not rely on it is because I don't want to pull such a huge dependency at the top of the dependency tree.
